### PR TITLE
Remove markdown elements from PM

### LINKED ISSUE #11027: markdown should be parsed in Private Message (PM) previews
As reported by @ravenlune (9272ab2c-4b66-4b74-aebd-799725885d1c):

"Markdown doesn't render in PM previews, when it does in the actual message being previewed."

image

I.e., markdown parsing needs to be done in the preview page on the left-hand side of the PM modal.

### PR DESCRIPTION
**Pull request by HydeHunter2**
Fixes #11027

### Changes
- Remove markdown elements from Private Message

Before | After
--------|--------
Git | Git

----
UUID: af1477b6-509a-4611-b851-d7a8e3a9c02a

### CODE REVIEW

**Comment by HydeHunter2** [comment]
~~Could you put a label `needs work` for this PR? (or just don't make a review yet)
I looked at the project in Zeplin and realized that it's necessary to limit the length of the text not by number of characters (since they are of different lengths), but somehow limit the number of lines.
I don’t know how to implement it yet, but I'll try (maybe CSS will help)~~

**Comment by HydeHunter2** [comment]
I did it!
~~*But without ellipsis at the end. No idea how to implement it.~~

**Comment by HydeHunter2** [comment]
###### Don't pay attention to past messages.
Now the length of the text is limited to 2 lines and there is ellipsis at the end.
Github

File: website/client/components/userMenu/inbox.vue [hunk 1]
```diff
@@ -209,6 +209,16 @@
     color: $gray-200;
     margin-bottom: 0.5rem;
   }
+
+  .messagePreview {
+    display: block;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
```
Comment (paglias) [inline website/client/components/userMenu/inbox.vue:216]: can you remove the -webkit prefixes from rule names? it'll be added automatically at compile time

Also it doesn't look like firefox and IE11 support it... is it really needed? @Tressley 

I'm wondering if we could get the same result by using `overflow: hidden;` and setting the height of the box given that we know the height of each line

**Comment by paglias** [comment]
Left one comment but it's working well as usual @HydeHunter2 :)

I'll put this up for testing as well

File: website/client/components/userMenu/inbox.vue (hunk 1 above)
Comment (HydeHunter2) [reply website/client/components/userMenu/inbox.vue:216]: I've read that `-webkit` is not supported by some browsers, but I found a similar solution and thought it's normal.

https://github.com/HabitRPG/habitica/blob/4289becccc8663985736094e9fa2360d44733f0c/website/client/components/challenges/challengeItem.vue#L45-L60

But if you say this isn't a good solution, I'll fix up it.
(But I still don't know how to implement an ellipsis using `overflow: hidden;` :( If it(ellipsis)'s definitely necessary, I'll try again)

Comment (paglias) [reply website/client/components/userMenu/inbox.vue:216]: Ah, i didn't noticed it was used elsewhere , in that case it's fine as long as we make sure the layout isn't too broken on Firefox for example

Comment (Tressley) [reply website/client/components/userMenu/inbox.vue:216]: Sorry, just getting a chance to answer a few pings; I do think it's necessary for this to work in Firefox, but not IE.

**Comment by Tressley** [comment]
This looks good to me 👍. There was a part of me that thought we should still show emojis in the preview, but I think the current implementation is a good introduction @HydeHunter2 . We can run with this and add emoji support in the future if we hear feedback.

**Comment by paglias** [comment]
> This looks good to me 👍. There was a part of me that thought we should still show emojis

Ah, you're right! I forgot about it but yeah it would be a big change from the current implementation so maybe we can revisit this later when we go through the other emojis issues that we have?

**Pull request merged**